### DISCUSSION TITLE
Compatibility with React 18?

### DISCUSSION THREAD

**Discussion by Nemmcy** [adobe/react-spectrum#3003] [General]
Components are not compatible with React 18 as it can be seen in package-lock.json:
```
"peerDependencies": {
    "react": "^16.8.0 || 17.x",
    "react-dom": "^16.8.0 || 17.x"
}
```
Eslint throws warnings, but shows 0 vulnerabilities and I haven't had any issue yet.

Is it safe to use React Aria with React 18? When will you update the libraries? Thank you in advance!

**Comment by devongovett** [comment]
See #2800. I'm not sure if we have tested recently. If you are trying it out, it would be great if you could comment on that issue with any problems you run into. We know there are issues with strict mode #779.
